### PR TITLE
Create mossdapp.toml

### PR DESCRIPTION
**Pull request by wenzhenxiang**

### CODE REVIEW

File: data/ecosystems/m/mossdapp.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Ecosystem Level Information
+title = "Moss Dapp"
+
+sub_ecosystems = []
+
+github_organizations = [https://github.com/mossdapp]
```
Comment (jubos) [inline data/ecosystems/m/mossdapp.toml:6]: Need quotes here.

**Review by jubos — CHANGES_REQUESTED** [review]
Looks good. Just needs some syntax fixes and can you add mossdapp to the appropriate ecosystem you are building in.  I assume Starknet.

File: data/ecosystems/m/mossdapp.toml (hunk 1 above)
Comment (wenzhenxiang) [reply data/ecosystems/m/mossdapp.toml:6]: thanks

**Review by eherrerosj — APPROVED** [review]
LGTM now, thank you!

**Pull request merged**